L/libjulia/libjulia@1.5/build_tarballs.jl

```diff
@@ -1,2 +1,2 @@
 include("../common.jl")
-build_julia(ARGS, v"1.5.3")
+build_julia(ARGS, v"1.5.4")
```

Comment:
One thing is not clear to me: will this be installed for Julia v1.5.0-1.5.3?  Because v1.5.4 broke ABI compatibility with previous versions in the 1.5 series

Reply:
That's why I initially put 1.5.4 in a separate dir, but as @fingolfin pointed out this package is only used as a build dependency, so it's up to the JLLs using this to set their compat correctly, depending on the version of libjulia they build against.